src/main/java/org/jenkinsci/plugins/github_branch_source/DefaultGitHubNotificationStrategy.java

```diff
@@ -8,16 +8,21 @@
 import org.jenkinsci.plugins.displayurlapi.DisplayURLProvider;
 import org.kohsuke.github.GHCommitState;
 
-public class GitHubBuildStatusNotificationStrategy {
-    protected Job<?, ?> job;
-    protected Run<?, ?> build;
-    protected TaskListener listener;
-    protected SCMSource source;
-    protected SCMHead head;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
-    public GitHubBuildStatusNotificationStrategy() {}
+/**
+ * @since TODO
+ */
+public class DefaultGitHubNotificationStrategy extends AbstractGitHubNotificationStrategy {
+    public DefaultGitHubNotificationStrategy() {}
 
-    protected String generateContext() {
+    /**
+     * @since TODO
```

Comment:
Normally we'd leave @since tags off where it is the same as the class the method is defined in. We can tidy up when merging if you don't get a chance to fix